composer.json

```diff
@@ -1,16 +1,25 @@
 {
     "name":        "hostnet/asset-bundle",
     "type":        "symfony-bundle",
-    "description": "Bundle to glue Assets",
+    "description": "Hostnet asset bundle allows for configuring the hostnet/asset-lib from within your symfony app",
```

Comment:
`Allows the configuration of the hostnet/asset-lib package in your Symfony application.`